docs/getting-started/deployment.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Deployment Profiles
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TabsConstants from '@site/core/TabsConstants';
+
+Deployment profiles provide 2 features:
+
+* "Defaults" provide preference values that are applied on first run (or after a factory reset).
+* "Locked" settings allow an administrator to pin preference values.
+
+They can be specified both by an "admin" or by the "user". If either the "defaults" or the "locked" settings exists in the "admin" context, then the "user" profile is ignored.
+
+### Preferences values at startup
+
+Rancher Desktop settings are determined as follows:
+
+* Load "admin" deployment profile (both "defaults" and "locked")
+* If neither of them exist then load "user" deployment profile (again both "defaults" and "locked")
+* Load saved preferences from `settings.json` file
+* If there are no saved settings, use the "defaults" profile loaded earlier instead
+* If the "defaults" profile didn't exist either, show the first-run dialog
+* Otherwise set any missing values to the builtin application defaults
+* Copy values from the "locked" profile over the current settings
+
+The user cannot modify any settings (via GUI or CLI) that have been locked by the profile.
+
+Rancher Desktop will refuse to load the application if a profile exists, but cannot be parsed correctly.
+
+Deployment profiles will not be modified or removed by Rancher Desktop. They will not be affected by a factory reset or uninstall.
+
+The structure of the profile data matches the application settins:
```

Comment:
Typo: `settings`

Reply:
@sunilarjun I'll create another PR based on Eric's feedback

Reply:
Sounds good, thank you!